lib/sidekiq_unique_jobs/client/extensions.rb

```diff
@@ -0,0 +1,14 @@
+module SidekiqUniqueJobs
+  module Client
+    module Extensions
+      def lock_queue_script
+        <<-LUA
+          local ret = redis.call('GET', KEYS[1])
+          if not ret or ret == 'scheduled' then
```

Comment:
This would have to change as well then with a composite key of some sort. Even leaving it as it was with just a simple integer. Not sure what happens in an application with a billion scheduled keys. The jit was needed to prevent unlocking jobs you shouldn't be able to unlock?